aten/src/ATen/native/cpu/LinearAlgebraKernel.cpp

```diff
@@ -79,8 +81,48 @@ void addr_kernel(TensorIterator &iter,
   );
 }
 
+template <typename scalar_t, typename acc_t=typename scalar_value_type<scalar_t>::type>
+void linalg_vector_norm_kernel_cpu_impl(TensorIterator& iter, Scalar ord) {
+  double ord_val;
+  if (ord.isFloatingPoint()) {
+     ord_val = ord.to<double>();
+  } else {
+     AT_ERROR("linalg_vector_norm_kernel_cuda_impl expects ord to be float");
```

Comment:
`TORCH_CHECK(false, ...`

`AT_ERROR` is deprecated.

Reply:
If this error can be triggered by user input then the name should be the user-facing name, too:

"linalg.vector_norm expects ..."